src/sidebar/store/modules/annotations.js

```diff
@@ -250,6 +240,7 @@ function addAnnotations(annotations, now) {
       currentAnnotationCount: getState().annotations.annotations.length,
     });
 
+    // If we're not in the sidebar, we're done here.
     if (!getState().viewer.isSidebar) {
       return;
     }
```

Comment:
FWIW, I think that some of the logic in the rest of this method (to do with anchoring timeouts) might be ideally factored out somewhere; I didn't tackle that in this body of work.